doc/userguide/rules/transforms.rst

```diff
@@ -188,3 +188,29 @@ Example::
 
     alert http any any -> any any (msg:"HTTP ua only"; http.header_names; \
        bsize:16; content:"|0d 0a|User-Agent|0d 0a 0d 0a|"; nocase; sid:1;)
+
+from_base64
```

Comment:
I think the modes should be explained

Reply:
@victorjulien I don't think `relax` has any meaning so I'd like to remove it.

I think the default mode should be `strict` instead of (the email/mime focused rfc4648).

Thoughts?

Reply:
I'm going to:
- Remove relax (no meaning)
- Change the default to `strict`